### ISSUE TITLE
Borg laws out of order

### ISSUE THREAD

**Issue by Titian3** [space-wizards/space-station-14#24354]
## Description
Had 3 borgs with laws out of order this round, it messes up the logic of the set with them being out of order. See screenshot.

**Reproduction**
I went in to a new borg as a mii to test
- Being made or spawning in as borg
- check laws

**Screenshots**
![image](https://github.com/space-wizards/space-station-14/assets/47093363/d9f0c3eb-dcaf-4243-99e8-81a4b0ab2562)

**Additional context**
NA

**Comment by deltanedas** [comment]
thats ion storm

**Comment by GreyMario** [comment]
>went into a borg as a mii

I know this is a typo, but now I just want to go to Science and play the Make-a-Mii song in Robotics for a full shift.

**Comment by Golinth** [comment]
> thats ion storm

I dont think so. Last couple rounds on Lizard have had backwards borg laws. Law 1 is always protect your own existence. Leads to... interesting borg dynamics.

**Comment by GreyMario** [comment]
Current code appears to be that this literally shouldn't be possible _without an ion storm,_ except for the sitaution in which the server is running a modified silicon-laws.yml. And I don't see why they'd set Lizard to this, so this is definitely ion storms.

**Comment by XxSWAG-MASTER-OVERLORDxX** [comment]
I'm not going to lie, this is actually a positive change for borgs, bug or not. it's fun and lets borgs not be round removed and defend themselves accordingly. 
I think the laws should stay like this, maybe reworded slightly.

**Comment by K-Dynamic** [comment]
Would be interesting if we could rearrange their laws yes

**Comment by XxSWAG-MASTER-OVERLORDxX** [comment]
because my understanding of the "bugged" laws is you cant harm crew unless you are told to by crew

**Comment by GoodWheatley** [comment]
This was an ion storm.

**Comment by Titian3** [comment]
> This was an ion storm.

@GoodWheatley - This was not Ion storm, logs from the round indicate no ion storm and in addition when I tested it I was built in to a new borg body so the ion storm should not have affected it:
<img width="608" alt="image" src="https://github.com/space-wizards/space-station-14/assets/47093363/70e7d550-8342-4cb0-8793-97efd9971a8e">

Just as a reference of an actual ion storm happening log wise.
![image](https://github.com/space-wizards/space-station-14/assets/47093363/7e4451f4-f1a8-4d5c-89e1-fd819f97e8f7)

Although it appears to be some transient issue so I will just raise an issue again when/if people aHelp it once more.

**Comment by EmoGarbage404** [comment]
wasn't event logging only just added recently?

**Comment by GoldenCan** [comment]
I have seen something like this whilst playing AI, ("Don't harm crew" was law 1, "Obey orders" was below, but it was also labeled as law 1, "Protect yourself" was labeled as law 2) so i think there is a bug where reordering of laws, probably because of ion storms, affects later cyborgs that use the same lawset. It appears to only affect the order of the laws, not remove or add anything.
